azure-mgmt-website/src/main/java/com/microsoft/azure/management/website/CountryPhoneCode.java

```diff
@@ -0,0 +1,291 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ */
+
+package com.microsoft.azure.management.website;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * Defines values for VirtualMachineSizeTypes.
+ */
+public final class CountryPhoneCode {
+    // CHECKSTYLE IGNORE Javadoc FOR NEXT 236 LINES
+    public static final CountryPhoneCode Albania = new CountryPhoneCode("+355");
+    public static final CountryPhoneCode Algeria = new CountryPhoneCode("+213");
+    public static final CountryPhoneCode Andorra = new CountryPhoneCode("+376");
```

Comment:
all caps - i know this is auto-gen'd - don;'t we have control over capitalization of extensible enums?